presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraClientModule.java

```diff
@@ -97,8 +96,7 @@ public static CassandraSession createCassandraSession(
         requireNonNull(config, "config is null");
         requireNonNull(extraColumnMetadataCodec, "extraColumnMetadataCodec is null");
 
-        Cluster.Builder clusterBuilder = Cluster.builder()
-                .withProtocolVersion(ProtocolVersion.V3);
+        Cluster.Builder clusterBuilder = Cluster.builder();
```

Comment:
I understood the problem why it was intentionally done. I could not find any API method in the driver to check the Cassandra version, instead found protocol version. 

Cassandra version older than 2.2 supports protocol version 3 and doesn't support for predicate IN. Initially, I changed only one java file for predicate pushdown but to support Cassandra version 2.1(IN clause not supported), need to changed other 6 files. Do you have any recommendation to handle this?